public/css/common.less

```diff
@@ -328,3 +328,93 @@ div.show-more {
 .servicegropup-list > col.object-statistics-graph {
   padding-right: .25em;
 }
+
+.comment-popup {
+  font-size: 1em / .857em; // // default font size / footer font size = 12px
+  height: 6em;
+  border-radius: 0.25em;
+  border: @gray-light 1px solid;
+  background-color: @body-bg-color;
+
+  .comment-list .main {
+    border: none;
+
+    // This is necessary to limit the visible comment lines
+    // because the popup is shown in detailed list mode only
+    .caption {
+      height: 3em;
+    }
+  }
+}
+
+li.list-item .status-icons {
+  display: flex;
+  align-items: center;
```

Comment:
Please add this where the general style/layout rules are for the `.status-icons`.